pom.xml

```diff
@@ -13,7 +13,7 @@
   <artifactId>sample-status-updater-cli</artifactId>
   <version>1.4.0-SNAPSHOT</version>
```

Comment:
NIT: Why is this a Snapshot version? 😕

Reply:
probably because maven removed the snapshot already automated here? 🤷🏼 
do you want me to update the versioning here?

Reply:
leave it, the CI takes over. no need to touch this anymore